enterprise_access/apps/api/v1/tests/test_views.py

```diff
@@ -1711,7 +1711,7 @@ def test_list_policies_for_enterprise(self):
 
     def test_list_without_group_uuid_query_param(self):
```

Comment:
test name had group_uuid in it, can we replace it with enterprise_uuid?

Reply:
good catch.

Reply:
done